identities/models.py

```diff
@@ -183,7 +183,8 @@ def handle_optout(sender, instance, created, **kwargs):
         sender=None,
         event_name='optout.requested',
         payload={
-            'details': identity.details,
+            'identity_id': str(identity.id),
```

Comment:
elsewhere in the system, we've just used `identity` to refer to the ID. I'd prefer that for consistency.